tox

```diff
@@ -12,13 +12,6 @@
 # canonical ${BASH_SOURCE} string global, reliably providing the absolute
 # pathnames of this script and hence this script's directory.
 #
-# --------------------( CAVEATS                            )--------------------
-# *THE HIGHER-LEVEL "tox" SCRIPT SHOULD TYPICALLY BE RUN INSTEAD.* This
-# lower-level script only exercises this project against the single Python
```

Comment:
I think this was an accidental copy paste. I just saw happened to see it scrolling by.

Reply:
*...lol.* Your all-seeing eye is all-seeing, indeed. That's *probably* a years-old copy-paste at this point. We have cats that are younger. Thanks so much for removing that ancient cruft – and this entire PR, really. You're stunningly rad, @frrad. :cat2: